src/js/actions/documents.js

```diff
@@ -307,13 +307,23 @@ define(function (require, exports) {
     /**
      * Initialize document and layer state, emitting DOCUMENT_UPDATED.
      *
-     * @return {Promise.<{currentIndex: number, docCount: number}>}
+     * @return {Promise.<{activeDocumentID: number, openDocumentIDs: Array.<number>}=>}
      */
     var initActiveDocument = function () {
-        return descriptor.getProperty("application", "numberOfDocuments")
+        var appRef = {
```

Comment:
Yes, very true. Even better perhaps would be to fix `descriptor.multiGetProperty` so that it correctly wraps an `"application"` reference like `descriptor.get` does.